portfolio/src/main/webapp/index.html

```diff
@@ -14,5 +14,42 @@ <h1>My Portfolio</h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+
+    <button class="open-button" id = "myButton" onclick="openForm()">Open Form</button>
+    <div id="myForm" class="form">
+        <div class="form-popup" id="myForm">
+            <span class="close" onclick="closeForm()">&times;</span>
+            <form action="/action_page.php" class="form-container">
```

Comment:
What's this php file?

If you want to use the "action" attribute, then it should probably be a POST to a java servlet.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

Or if you want to use javascript, then I think the Submit button needs an onclick atribute, but then the action attribute should probably be blank.

Reply:
Im not quite sure what route I want to take yet. I probably need to discuss with Adetunji about how he wants to connect the form to the backend.

Reply:
the name of the servlet is "/marker". The data goes there.